### ISSUE TITLE
Feature Request: Multi-User Support in CasaOS

### ISSUE THREAD

**Issue by RizWali** [IceWhaleTech/CasaOS#1789]
**Feature Request: Multi-User Support in CasaOS**

**Is your feature request related to a problem? Please describe.**
Currently, CasaOS only supports a single admin user, which limits flexibility and collaboration within the system. This lack of multi-user support poses challenges for households or organizations where multiple individuals need access to CasaOS with varying levels of permissions.

**Describe the solution you'd like**
I would like to request the addition of multi-user support in CasaOS, allowing for the creation of standard user accounts in addition to the existing admin account. Standard user accounts should have limited permissions, enabling users to interact with CasaOS while restricting their ability to modify critical settings or configurations. Admin users should retain full control over the system and have the ability to manage user accounts.

**Describe alternatives you've considered**
1. **Shared Admin Credentials:** Users could share the admin credentials, but this approach presents security risks and lacks accountability.
2. **Workarounds with Guest Accounts:** Users might utilize guest accounts or temporary logins, but these solutions are not tailored to the needs of multiple permanent users.

**Additional context**
- Multi-user support will enhance collaboration and convenience for households, businesses, or organizations using CasaOS.
- Each user should have a unique login and password for personalized access.
- Admin users should be able to assign and manage permissions for standard users, ensuring security and control over the system.

**Comment by Neso2** [comment]
yes indeed that would be very important. Multi user accounts are really missing.

**Comment by GoodLittleDev** [comment]
Currently you can add users to the db and there is a roles column but there is no list of roles, and so all users are admin. (does let you add additional users with unique logins)
